features/view_dishes.feature

```diff
@@ -4,14 +4,22 @@ Feature: List dishes on landing page
   I would like to view dishes on a webpage
 
   Background:
-    Given the following dishes exist:
-      | name         | description             | price | min_quantity |
-      | Dish 1       | Description for Dish 1  | 100   |  10          |
-      | Dish 2       | Description for Dish 2  | 200   |  10          |
-      | Dish 3       | Description for Dish 3  | 300   |  10          |
+    Given the following categories exist:
+      | name    | description                | sort_key |
```

Comment:
Wouldn't be better if the name be sort_categories, since we/GigaFood probably would like other sorting keys for dishes and maybe ingredients.

Reply:
This is a sort_key in the category so this sort_key belongs to the category.